test/test-etsi.c

```diff
@@ -0,0 +1,147 @@
+/*
```

Comment:
This is an old copyright message from our GPLv2 days -- ideally this should be an LGPL one.

Would also be a good idea to update the copyright year to 2021 ;)